lib/pretty/style.ml

```diff
@@ -1,15 +1,44 @@
-type t = ANSITerminal.style list
-
-let fg_black = ANSITerminal.[ Foreground Black ]
-let fg_white = ANSITerminal.[ Foreground White ]
-let repo = ANSITerminal.[ Bold; blue ]
-let selected = ANSITerminal.[ Bold; green ]
-let directory = ANSITerminal.[ Bold; magenta ]
-let chosen = ANSITerminal.[ Bold; magenta ]
-let secondary = ANSITerminal.[ cyan ]
-let bold = ANSITerminal.[ Bold ]
-let issue_open = ANSITerminal.[ green ]
-let issue_closed = ANSITerminal.[ red ]
-let pr_closed = ANSITerminal.[ Bold; red ]
-let pr_open = ANSITerminal.[ Bold; green ]
-let pr_merged = ANSITerminal.[ Bold; magenta ]
+module Color = Ansifmt.Color
+module Styling = Ansifmt.Styling
+
+type t = Styling.t
+
+let ( & ) = Styling.( & )
```

Comment:
I reexport things from `Ansifmt` to follow something kinda Domain-Driven Design and provide a small local wrapper around `ansifmt` so other modules inside GitHub TUI shouldn't depend on Ansifmt directly, only on relevant parts.